locales/uk/common.json

```diff
@@ -72,8 +72,11 @@
 	"password.reset.confirmation": "На вказану електронну пошту був надісланий е-мейл з інструкціями як поновити пароль",
 	"password.reset.error": "Помилка в надісланні анкети",
 	"nabidka.title": "Тут напишіть свою пропозицію",
+	"nabidka.addAnother": "",
```

Comment:
rather use google translator than  empty string (and contact zdenek for later improvement)